src/hooks/useReduxContext.js

```diff
@@ -18,8 +19,8 @@ import { ReactReduxContext } from '../components/Context'
  *   return <div>{store.getState()}</div>
  * }
  */
-export function useReduxContext() {
-  const contextValue = useContext(ReactReduxContext)
+export function useReduxContext(context = ReactReduxContext) {
```

Comment:
I'm not sure there's much value in making this change. It's essentially a useContext + invariant. I'd just leave this as-is.